src/App.tsx

```diff
@@ -1,10 +1,56 @@
 /* eslint-disable jsx-a11y/control-has-associated-label */
-import React from 'react';
+import React, { useEffect, useState } from 'react';
+import cn from 'classnames';
+import { Todo } from './types/Todo';
+import { getTodos } from './api/todos';
+import { TodoList } from './components/TodoList/TodoList';
+import { FilteredBy } from './types/FilteredBy';
+import { TodoFilter } from './components/TodoFilter/TodoFilter';
 import { UserWarning } from './UserWarning';
 
-const USER_ID = 0;
+const USER_ID = 10348;
 
 export const App: React.FC = () => {
+  const [todos, setTodos] = useState<Todo[]>([]);
+  const [filteredTodos, setFilteredTodos] = useState<Todo[]>([]);
+  const [filteredBy, setfilteredBy] = useState(FilteredBy.ALL);
```

Comment:
The state setter 'setfilteredBy' should be camelCase, change it to 'setFilteredBy'.